sql/2019/17_CDN/17_04.sql

```diff
@@ -0,0 +1,25 @@
+#standardSQL
+# 17_04: Percentiles of header volume
+SELECT
+  client,
+  ROUND(APPROX_QUANTILES(reqheader, 1000)[OFFSET(100)] / 1024, 2) AS p10_reqheader,
```

Comment:
All of the results are < 1 so I'd recommend removing the `/ 1024` and rounding for each of these.
```suggestion
  APPROX_QUANTILES(reqheader, 1000)[OFFSET(100)] AS p10_reqheader,
```